src/transformers/models/nllb/tokenization_nllb_fast.py

```diff
@@ -177,17 +174,15 @@ def __init__(
             cls_token=cls_token,
             unk_token=unk_token,
             pad_token=pad_token,
-            mask_token=mask_token,
             src_lang=src_lang,
             tgt_lang=tgt_lang,
-            additional_special_tokens=_additional_special_tokens,
+            mask_token=mask_token,
+            additional_special_tokens=additional_special_tokens,
             legacy_behaviour=legacy_behaviour,
             **kwargs,
         )
 
-        self.vocab_file = vocab_file
-
-        self.lang_code_to_id = {
+        self._lang_code_to_id = {
             lang_code: self.convert_tokens_to_ids(lang_code) for lang_code in FAIRSEQ_LANGUAGE_CODES
```

Comment:
Here the comment is the same: currently, if I change the list of languages, I cannot override this line. 
But if you use the FAIRSEQ_LANGUAGE_CODES only as the default value for the `additional_special_tokens` argument, it will be no longer the case, and the users will be able to use tokenizers for custom language sets.

Reply:
You should not use this dictionary anymore:
```python
    @property
    def lang_code_to_id(self):
        logger.warning_once(
            "the `lang_code_to_id` attribute is deprecated. The logic is natively handled in the `tokenizer.adder_tokens_decoder`"
            " this attribute will be removed in `transformers` v4.38"
        )
        return self._lang_code_to_id

```